### PR TITLE
add CrowdStrikeStreamingPreProcessing

### PR DESCRIPTION
**Pull request by meirwah**

### CODE REVIEW

File: Scripts/script-CrowdStrikeStreamingPreProcessing.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          res = True
+      else:
+          otherIncidents = sameIncidents[0]['Contents']['data']
+          entries = []
+          entries.append({'Contents': "Duplicate incident from crowdstrike: "+inc['name']})
+          entries.append({"Type": 1, "ContentsFormat": "json", "Contents": json.dumps(inc)})
+          entries_str = json.dumps(entries)
+          demisto.executeCommand("addEntries", {"id":otherIncidents[0]["id"],"entries":entries_str})
+
+  demisto.results(res);
```
Comment (slavikm) [inline Scripts/script-CrowdStrikeStreamingPreProcessing.yml]: Remove `;`

File: Scripts/script-CrowdStrikeStreamingPreProcessing.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+commonfields:
+  id: CrowdStrikeStreamingPreProcessing
+  version: -1
+name: CrowdStrikeStreamingPreProcessing
+script: |-
+  import sys
+
+  res =False
```
Comment (slavikm) [inline Scripts/script-CrowdStrikeStreamingPreProcessing.yml]: Space

File: Scripts/script-CrowdStrikeStreamingPreProcessing.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  for inc in demisto.incidents():
+      host = ''
+      for t in inc['labels']:
+          if t['type'] == 'System':
+              host = t['value']
+      sameIncidents = demisto.executeCommand("getIncidents",
```
Comment (slavikm) [inline Scripts/script-CrowdStrikeStreamingPreProcessing.yml]: If there is no host - no need to search incidents, I think

File: Scripts/script-CrowdStrikeStreamingPreProcessing.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+          {"query":"labels.value:\""+host+"\"  and labels.type:System"})
+      # if found sameIncidents found, add this incident data to war room
+      sameIncidentsCount = sameIncidents[0]['Contents']['total']
```
Comment (slavikm) [inline Scripts/script-CrowdStrikeStreamingPreProcessing.yml]: Do you always have `total`? Just wondering if you'll have key error if for some reason you have an error. I think you should check for errors here.

Comment (meirwah) [reply Scripts/script-CrowdStrikeStreamingPreProcessing.yml]: even if error.. not much I can do.. it's preprocessing nothing to return to user

File: Scripts/script-CrowdStrikeStreamingPreProcessing.yml [hunk 5]
```diff
@@ -0,0 +1,39 @@
+commonfields:
+  id: CrowdStrikeStreamingPreProcessing
+  version: -1
+name: CrowdStrikeStreamingPreProcessing
+script: |-
+  import sys
+
+  res=False
```
Comment (slavikm) [inline Scripts/script-CrowdStrikeStreamingPreProcessing.yml]: Default should be true and turn to false only if you find dup. Also, add back spaces :-)

**Comment by meirwah** [comment]
@slavikm can you review again?

File: Scripts/script-CrowdStrikeStreamingPreProcessing.yml [hunk 6]
```diff
@@ -0,0 +1,40 @@
+commonfields:
+  id: CrowdStrikeStreamingPreProcessing
+  version: -1
+name: CrowdStrikeStreamingPreProcessing
+script: |-
+  import sys
```
Comment (orius123) [inline Scripts/script-CrowdStrikeStreamingPreProcessing.yml]: Do you need to import sys?

**Comment by meirwah** [comment]
@orius123 plz review again

**Pull request merged**